packages/flutter_tools/lib/src/commands/devices.dart

```diff
@@ -164,3 +182,100 @@ class DevicesCommandOutput {
     );
   }
 }
+
+const String _checkingForWirelessDevicesMessage = 'Checking for wireless devices...';
+const String _noAttachedCheckForWireless = 'No devices found yet. Checking for wireless devices...';
+const String _noWirelessDevicesFoundMessage = 'No wireless devices were found.';
+
+class MacOSDevicesCommandOutput extends DevicesCommandOutput{
+  MacOSDevicesCommandOutput({
```

Comment:
Inject the globals objects you're using into the constructor, we've been trying to get rid of `globals.dart` for years at this point...

https://github.com/flutter/flutter/issues/47161

```suggestion
  MacOSDevicesCommandOutput({
    required Logger logger,
    required DeviceManager deviceManager,
```
etc

Reply:
Good to know, I'll keep that in mind for the future too. Done.